.github/workflows/md-link-checker.yml

```diff
@@ -1,6 +1,9 @@
 name: Check Markdown links
 on:
+  pull_request:
```

Comment:
If you add the same `paths: [**.*.md]` after this field, this would prevent the action from being run on PRs that don't change md files right ?

Reply:
I believe so:

```yaml
on:
    pull_request:
        paths: 
            - "**.*md"
    push:
        branches
            - main
```

Do you mean like this?

Reply:
I meant this: 

```yaml
on:
    pull_request:
        paths: 
            - "**.*md"
    push:
        paths: 
            - "**.*md"        
        branches:
            - main
```

Not a github action expert but I think this ensures the action is run only when md files are modified, wether from PR or from pushes on main.